services/user.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email=None,
+                first_name=None,
+                last_name=None
+                ) -> User:
+    user = get_user_model().objects.create_user(username=username)
+    user.set_password(password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()

```

Comment:
You are calling `save` method here which is not necessary. `create_user` method already saves the user object in the database. You only need to call `save` method if you have made changes to the user object after calling `create_user` method. This is a code quality issue.